common/app/model/facia/PressedCollection.scala

```diff
@@ -64,6 +66,13 @@ case class PressedCollection(
     copy(hasMore = hasMore)
   }
 
+  def fullDCR(visible: Int): PressedCollection = {
+    val liteCurated = curated.take(visible)
+    val liteBackfill = backfill.take(visible - liteCurated.length)
+    val hasMore = curatedPlusBackfillDeduplicated.length > visible
+    copy(hasMore = hasMore, liteCuratedLength = Some(liteCurated.length), liteBackfillLength = Some(liteBackfill.length))
```

Comment:
I think we can have:

```
liteCuratedLength = liteCurated.length, 
liteBackfillLength = liteBackfill.length
```

`liteCurated` and `liteBackfill` are Lists. If they're empty, their length will be 0.